test/integration/download-known-assets.js

```diff
@@ -82,12 +101,11 @@ test('load', t => {
     for (var i = 0; i < testAssets.length; ++i) {
         const assetInfo = testAssets[i];
 
-        const promise = storage.load(assetInfo.type, assetInfo.id);
+        var promise = storage.load(assetInfo.type, assetInfo.id, assetInfo.ext);
```

Comment:
This repo still uses Node 4, which is OK with `const` but not `let` in this context.